deeplearning/ml4pl/poj104/dataset.py

```diff
@@ -789,12 +823,14 @@ def raw_file_names(self):
 
     @property
     def processed_file_names(self):
-        base = f'{self.split}_data.pt'
+        name = f"{self.split}_cdfg" if self.cdfg else self.split
+
+        base = f'{name}_data.pt'
```

Comment:
Ah great! That's exactly what I wanted to do and where I was worried we'll get undefined behaviour.